config/common.go

```diff
@@ -4,6 +4,7 @@
 package config
 
 import (
+	"github.com/mattermost/mattermost-server/utils"
```

Comment:
nit: Can you apply [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to this file? (We should really make this part of the CI). The almost-idiomatic way of ordering imports is something like:
```
<stdlib>

<external to this repository>

<internal to this repository>
```

`goimports` is basically `go fmt` + fixing that ordering above. I have it integrated into my editor to make it automatic.

Reply:
Awesome, mine is sent up to do that now too.